libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/user/MatrixUser.kt

```diff
@@ -16,10 +16,13 @@
 
 package io.element.android.libraries.matrix.api.user
 
+import android.os.Parcelable
 import io.element.android.libraries.matrix.api.core.UserId
+import kotlinx.parcelize.Parcelize
 
+@Parcelize
```

Comment:
It feels a bit weird to make this `Parcelable`, but I guess it's fine since it's the Android SDK after all, not a pure Kotlin one? I think we discussed this in the past (and for this same class if I'm not mistaken) with @ganfra .

Reply:
Yeah, agree it's a bit weird.

We only need it now for rememberSaveable calls in compose, so we could instead implement a [Saver](https://developer.android.com/reference/kotlin/androidx/compose/runtime/saveable/Saver) in the `matrixui` module and use that whenever we call rememberSaveable with a MatrixUser. It's a bit prone to error, though, as it just results in a runtime crash if you pass a "non-savable" object to rememberSaveable. What do you think?

Reply:
Let's keep it `Parcelable`, as discussed in the room.